tests/test_service_env.py

```diff
@@ -247,3 +247,7 @@ def predict(self, df):
 
     assert 'test-dep-1' in env_yml['dependencies']
     assert 'bentoml-test-lib' in env_yml['dependencies']
+
+    assert os.path.isfile(
```

Comment:
Always test the actual behavior you are implementing or fixing - in this case, add a new example BentoService that uses the conda env file argument and test if it works as expected.